test/File.js

```diff
@@ -117,6 +117,71 @@ describe('File', function() {
       file.sourceMap.should.equal(sourceMap);
       done();
     });
+
+    it('should normalize path', function() {
+      var file = new File({ path: '/test/foo/../test.coffee' });
+
+      if (process.platform === 'win32') {
```

Comment:
going to poke at this and see if we can use `path.normalize` to avoid the `process.platform` test